packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -68,7 +73,7 @@ const CMSPage = props => {
     ) {
         content = (
             <Fragment>
-                <Title>{title}</Title>
+                <Title>{meta_title.length > 0 ? meta_title : title}</Title>
```

Comment:
```suggestion
                <Meta name="title" content={meta_title} />
```
But it looks like meta title seem to be on the way out https://moz.com/community/q/title-vs-meta-name-title-both-or-just-one-of-them.

Reply:
It looks as if Magento will always use the `meta_title` if present in both the `<title />` and `<meta name="title" />`. So I'll include the meta tag but use the same logic to determine which title to include.

<img width="552" alt="image" src="https://user-images.githubusercontent.com/1774374/91058279-e201cc80-e5ed-11ea-9de1-5745755536e0.png">